.gitmodules

```diff
@@ -47,3 +47,6 @@
 	path = src/tools/rustc-perf
 	url = https://github.com/rust-lang/rustc-perf.git
 	shallow = true
+[submodule "src/gcc"]
+	path = src/gcc
+	url = https://github.com/rust-lang/gcc.git
```

Comment:
Could you also please add `shallow = true` here? Otherwise it's very.. heavy to clone :laughing:

Reply:
Yes it is. ^^'